tests/run-make/panic-abort-eh_frame/rmake.rs

```diff
@@ -0,0 +1,23 @@
+// An `.eh_frame` section in an object file is a symptom of an UnwindAction::Terminate
+// being inserted, useful for determining whether or not unwinding is necessary.
+// This is useless when panics would NEVER unwind due to -C panic=abort. This section should
+// therefore never appear in the emit file of a -C panic=abort compilation, and this test
+// checks that this is respected.
+// See https://github.com/rust-lang/rust/pull/112403
+
+//@ ignore-i686-pc-windows-gnu
+// Reason: the DW_CFA symbol appears on this specific architecture
```

Comment:
Question: cc @Nilstrieb (since you were the reviewer of #112403) and @saethlin, since I lack context for this test: is it intentional that this property is only preserved for linux, but not necessarily for `i686-pc-windows-gnu`, or is this a bug? The original test was `only-linux` but I couldn't find reasons for *why* it had to be `only-linux` other than it was impacting RfL so obviously we would want to test it on linux.

Reply:
The context is that everything under -Zub-checks creates non-unwinding panics, so it is a problem for RFL and quite goofy in general if enabling -Zub-checks produces new sections which are only for unwinding.

I don't know enough about this target to say whether it has this section with UB checks disabled. If it has this section with UB checks off that seems like a separate bug. If it gains this section when UB checks are enabled on this target specifically I would really like to know why.

Reply:
Since the original Makefile test was *only* exercised on linux (`only-linux`), I think it's fine if we leave this ignore in with a FIXME linking to https://github.com/rust-lang/rust/issues/128136 which asks about this test in case this is *not* intended.

Reply:
Sounds good to me!